kolibri/locale/es_419/LC_MESSAGES/kolibri.core.default_frontend-messages.json

```diff
@@ -236,6 +236,7 @@
   "CommonCoreStrings.tasksLabel": "Tareas",
   "CommonCoreStrings.technicalAndVocationalTraining": "Formación técnica y profesional",
   "CommonCoreStrings.tertiary": "Enseñanza superior",
+  "CommonCoreStrings.timeSpentLabel": "Tiempo utilizado",
```

Comment:
This translation was missing so I put it manually because it gave me many errors in the browser